### DISCUSSION TITLE
Release issue: GoReleaser problem I can't find hint for

### DISCUSSION THREAD

**Discussion by michalfita** [michalfita/packer-plugin-cross#16] [Maintenance]
@dbast I pushed v1.1.0 tag and CI failed https://github.com/michalfita/packer-plugin-cross/actions/runs/7340316453/job/19986140955

```
  • building binaries
    • building                                       binary=dist/packer-plugin-cross_linux_riscv64/packer-plugin-cross_v1.1.0__linux_riscv64
    • building                                       binary=dist/packer-plugin-cross_linux_amd64_v1/packer-plugin-cross_v1.1.0__linux_amd64
    • building                                       binary=dist/packer-plugin-cross_linux_arm64/packer-plugin-cross_v1.1.0__linux_arm64
  ⨯ release failed after 0s                  error=failed to build for linux_arm64: exit status 1: 2023/12/27 17:00:50 unsupported GOARCH arm64
 ```
 which suggest Go is unable to cross-compile... however, from what I've read `arm64` should be handled by default.

**Comment by dbast** [comment]
This should fix it https://github.com/michalfita/packer-plugin-cross/pull/17 ... regarding the outdated actions: there is a depandabot config in the repo, but dependabot is disabled due to the repo being a fork... we should enable via https://github.com/michalfita/packer-plugin-cross/network/updates (don't have the permissions) to keeps things updated.

**Reply by michalfita** [reply]
I've just enabled the Dependabot.

**Reply by michalfita** [reply]
So, pushed `v.1.1.1` and release is still broken, however, different problem now:
```
 ⨯ release failed after 1m32s               error=scm releases: failed to publish artifacts: could not release: POST https://api.github.com/repos/michalfita/packer-plugin-cross/releases: 403 Resource not accessible by integration []
Error: The process '/opt/hostedtoolcache/goreleaser-action/1.23.0/x64/goreleaser' failed with exit code 1
```
Do we miss some tokens or something? Or should I just try to re-run the action, @dbast?

**Reply by michalfita** [reply]
@dbast thanks for you help - looks like it worked this time. 🕺🏼

**Comment by dbast** [comment]
by the way, the 1.1.0 container publish worked https://github.com/michalfita/packer-plugin-cross/pkgs/container/packer-plugin-cross :)
